index.html

```diff
@@ -11,15 +11,21 @@
     <header id="headline">
       <nav class="nav-container">
         <div class="logo-text"><a href="#">Spyda</a></div>
-        <div class="right-logo">
-          <img
-            src="images/menu-icon.png"
-            alt="Menu Icon for toggle on small screens"
-          />
+      <div class = "mobile-menu">
+        <ul class = "nav-menu">
+          <li class = "nav-item"><a href = "#headline">Portfolio</a></li>
```

Comment:
In the mobile menu, when I click on the portfolio link, it doesn't take me to the portfolio section but to the top of the page. Consider fixing it.